src/escpos/escpos.py

```diff
@@ -1433,23 +1434,22 @@ def writelines(self, text, **kwargs):
         # TODO flush? or on print? (this should prob rather be handled by the _raw-method)
         for line in lines:
             self.printer.set(**params)
-            if isinstance(text, six.text_type):
-                self.printer.text("{0}\n".format(line))
-            else:
-                self.printer.text("{0}\n".format(line))
+            self.printer.text("{0}\n".format(line))
```

Comment:
```suggestion
            self.printer.text(f"{line}\n")
```

Reply:
I'm not used to these new kind of f-string.

Reply:
In the end it is just a more compact form of the format syntax (without the boilerplate like this: `"{thing}".format(thing=thing)`)